provisioning/osc-install

```diff
@@ -106,6 +110,12 @@ setup_training_repo() {
   cd
 }
 
+# Usage: validate_ssh <hostname>
```

Comment:
This could be a good "library" function as I can see other areas where this could be used as well. Anyway; I'll go ahead and merge the change - just wanted to make this note and we can split it out with a future delivery.